README.md

```diff
@@ -1,49 +1,65 @@
+  - Automatic profile switching based on active window.
+  - Macros, button cycling, rapid fire, modeshift, mouse regions...
+  - Emulates Xbox360 controller, mouse, trackball and keyboard.
 
 Based on [Standalone Steam Controller Driver](https://github.com/ynsta/steamcontroller) by [Ynsta](https://github.com/ynsta).
 
-##### Like what I'm doing?
+## Like what I'm doing?
 
 [![Help me become filthy rich on Patreon](https://img.shields.io/badge/Help_me_become_filthy_rich_on-Patreon-Orange.svg)](https://www.patreon.com/kozec) <sup>or</sup> [![pay what you want with PayPal](https://img.shields.io/badge/pay_what_you_want_with-Paypal-yellow.svg)](https://www.paypal.com/cgi-bin/webscr?cmd=_donations&business=77DQD3L9K8RPU&lc=SK&item_name=kozec&item_number=scc&currency_code=EUR&bn=PP%2dDonationsBF%3abtn_donate_LG%2egif%3aNonHosted)
-##### Packages
-- Ubuntu (deb-based distros): in [openSUSE Build Service](http://software.opensuse.org/download.html?project=home%3Akozec&package=sc-controller).
-- Fedora, SUSE (rpm-based distros): in [openSUSE Build Service](http://software.opensuse.org/download.html?project=home%3Akozec&package=sc-controller).
-- Arch, Manjaro: in [AUR](https://aur.archlinux.org/packages/sc-controller-git/)
-- Solus: search for `sc-controller` in Software Center
-- Exherbo: in [hardware](https://git.exherbo.org/summer/packages/input/sc-controller)
-- Void Linux: `xbps-install -S sc-controller`
-
-##### To run without package
-- download and extract [latest release](https://github.com/kozec/sc-controller/releases/latest)
-- navigate to extracted directory and execute `./run.sh`
-
-##### To install without package
-- download and extract  [latest release](https://github.com/kozec/sc-controller/releases/latest)
-- `python2 setup.py build`
-- `python2 setup.py install`
-
-
-##### Dependencies
-- python 2.7, GTK 3.22 or newer and [PyGObject](https://live.gnome.org/PyGObject)
-- [python-gi-cairo](https://packages.debian.org/sid/python-gi-cairo) and [gir1.2-rsvg-2.0](https://packages.debian.org/sid/gir1.2-rsvg-2.0) on debian based distros (included in PyGObject elsewhere)
-- [setuptools](https://pypi.python.org/pypi/setuptools)
-- [python-pylibacl](http://pylibacl.k1024.org/) recommended
-- [python-evdev](https://python-evdev.readthedocs.io/en/latest/) strongly recommended
+
+## Packages
+
+### Ubuntu (deb-based distros):
+Found in [openSUSE Build Service](http://software.opensuse.org/download.html?project=home%3Akozec&package=sc-controller).
+
+### Fedora, SUSE (rpm-based distros):
+Found in [openSUSE Build Service](http://software.opensuse.org/download.html?project=home%3Akozec&package=sc-controller).
+
+### Arch, Manjaro (arch-based distros):
+Found in [AUR](https://aur.archlinux.org/packages/sc-controller-git/)
+
+### Solus:
+Search for `sc-controller` in Software Center or run `sudo eopkg it sc-controller` from a terminal.
+
+### Exherbo:
+Found in [hardware](https://git.exherbo.org/summer/packages/input/sc-controller)
+
+### Void Linux:
```

Comment:
Larger headers are probably fine, but listing distros like this makes it much less readable.

Reply:
I do not quite agree with you here, but I will do as you say :+1:

Reply:
Okay. I changed my mind, with some improvements it can look really good...